### ISSUE TITLE
Error trying yarn

### ISSUE THREAD

**Issue by 573** [DanielDe/org-web#130]
How can I get aroud this peer-dependencies error:

>[~/org-web]$ yarn
➤ YN0000: ┌ Resolution step
➤ YN0002: │ babel-preset-react-app@npm:7.0.2 doesn't provide webpack (p3a524), requested by babel-loader
➤ YN0002: │ org-web@workspace:. doesn't provide @babel/core (p38c75), requested by babel-loader
➤ YN0002: │ react-dev-utils@npm:8.0.0 doesn't provide typescript (p68b94), requested by fork-ts-checker-webpack-plugin
➤ YN0002: │ react-dev-utils@npm:8.0.0 doesn't provide webpack (pb44d3), requested by fork-ts-checker-webpack-plugin
➤ YN0000: │ Some peer dependencies are incorrectly met; run yarn explain peer-requirements <hash> for details, where <hash> is the six-letter p-prefixed code
➤ YN0000: └ Completed in 0s 293ms
➤ YN0000: ┌ Fetch step
➤ YN0000: └ Completed in 0s 681ms
➤ YN0000: ┌ Link step
➤ YN0007: │ org-web@workspace:. must be built because it never has been before or the last one failed
➤ YN0009: │ org-web@workspace:. couldn't be built successfully (exit code 243, logs can be found here: /run/user/1000/xfs-ee661d1f/build.log)
➤ YN0000: └ Completed in 1s 871ms
➤ YN0000: Failed with errors in 3s 43ms

Tried i. e.

```console
yarn dlx install-peerdeps -Y -o babel-preset-react-app
```
No fix.

**Comment by 573** [comment]
This describes a workaround:

_Originally posted by @DemianX0 in https://github.com/storybookjs/storybook/issues/11397
> Follow-up for 6.2.9: #14838 These warnings have been fixed pre-6.1.0 in [eaa9410](https://github.com/storybookjs/storybook/commit/eaa9410a4d8ea4a35bf9a58f7062842e15a0df78)
> 
> **Describe the bug** `yarn install` (Yarn 2 - berry) produces this output. These are warnings only, not blocking:
> 
> ```
> ➤ YN0000: ┌ Resolution step
> ➤ YN0002: │ @storybook/api@npm:5.2.8 doesn't provide react-dom@* requested by @storybook/router@npm:5.2.8
> ➤ YN0002: │ @storybook/api@npm:5.2.8 doesn't provide react-dom@* requested by @storybook/theming@npm:5.2.8
> ➤ YN0002: │ @storybook/html@npm:5.2.8 [b2d93] doesn't provide react@* requested by @storybook/core@npm:5.2.8
> ➤ YN0002: │ @storybook/html@npm:5.2.8 [b2d93] doesn't provide react-dom@* requested by @storybook/core@npm:5.2.8
> ➤ YN0002: │ @storybook/client-api@npm:5.2.8 doesn't provide react@* requested by @storybook/router@npm:5.2.8
> ➤ YN0002: │ @storybook/client-api@npm:5.2.8 doesn't provide react-dom@* requested by @storybook/router@npm:5.2.8
> ➤ YN0002: │ @storybook/ui@npm:5.2.8 doesn't provide @emotion/core@^10.0.27 requested by emotion-theming@npm:10.0.27
> ➤ YN0002: │ @storybook/core@npm:5.2.8 [55985] doesn't provide @babel/core@^7.0.0-0 requested by @babel/plugin-proposal-c
> lass-properties@npm:7.8.3
> ➤ YN0002: │ @storybook/core@npm:5.2.8 [55985] doesn't provide @babel/core@^7.0.0-0 requested by @babel/plugin-proposal-o
> bject-rest-spread@npm:7.9.6
> ➤ YN0002: │ @storybook/core@npm:5.2.8 [55985] doesn't provide @babel/core@^7.0.0-0 requested by @babel/plugin-syntax-dyn
> amic-import@npm:7.8.3
> ➤ YN0002: │ @storybook/core@npm:5.2.8 [55985] doesn't provide @babel/core@^7.0.0-0 requested by @babel/plugin-transform-
> react-constant-elements@npm:7.9.0
> ➤ YN0002: │ @storybook/core@npm:5.2.8 [55985] doesn't provide @babel/core@^7.0.0-0 requested by @babel/preset-env@npm:7.
> 9.6
> ➤ YN0000: └ Completed in 15.13s
> ➤ YN0000: ┌ Fetch step
> ➤ YN0000: └ Completed in 23.51s
> ➤ YN0000: ┌ Link step
> ➤ YN0000: └ Completed in 2.42s
> ➤ YN0000: Done in 3.07m
> ```
> 
> **Fixed in 6.1.0 [EDITED]**
> 
> The missing peerDeps were added in: https://github.com/storybookjs/storybook/releases/tag/v6.1.0-beta.1 [eaa9410#diff-3e4e211bc6c8ad1c4934045fbab3fac862e23c8e00915e433d09d7eed15a4851R49](https://github.com/storybookjs/storybook/commit/eaa9410a4d8ea4a35bf9a58f7062842e15a0df78#diff-3e4e211bc6c8ad1c4934045fbab3fac862e23c8e00915e433d09d7eed15a4851R49)
> 
> **Temporary fix for previous versions [EDITED]**
> 
> Create `.yarnrc.yml` next to package.json:
> 
> ```yaml
> packageExtensions:
>   "@storybook/addon-actions@*":
>     peerDependencies:
>       react-dom: "*"
>   "@storybook/addons@*":
>     peerDependencies:
>       react-dom: "*"
>   "@storybook/api@*":
>     peerDependencies:
>       react: "*"
>       react-dom: "*"
>   "@storybook/client-api@*":
>     peerDependencies:
>       react: "*"
>       react-dom: "*"
>   "@storybook/core@*":
>     peerDependencies:
>       "@babel/core": "^7.0.0-0"
>   "@storybook/html@*":
>     peerDependencies:
>       "@babel/core": "^7.0.0-0"
>       react: "*"
>       react-dom: "*"
>   "@storybook/ui@*":
>     peerDependencies:
>       emotion/core: "^10.0.27"
> ```
> 
> **To Reproduce** Steps to reproduce the behavior:
> 
>     1. [Install Yarn 2 berry](https://yarnpkg.com/getting-started/install) in a project using storybook
> 
>     2. Run: `yarn`
> 
> 
> **Expected behavior** No warning messages:
> 
> ```
> ➤ YN0000: ┌ Resolution step
> ➤ YN0000: └ Completed in 1.33s
> ➤ YN0000: ┌ Fetch step
> ➤ YN0000: └ Completed in 11.63s
> ➤ YN0000: ┌ Link step
> ➤ YN0000: └ Completed in 8.73s
> ➤ YN0000: Done in 21.91s
> ```
> 
> **Screenshots**
> 
> **Code snippets** Relevant part of package.json
> 
> ```
> "devDependencies": {
> 	"@babel/core": "7.7.7",
> 	"@storybook/html": "5.2.8",
> 	"@types/jquery": "3.3.33",
> 	"@types/mustache": "4.0.1",
> 	"@types/node-fetch": "2.5.7",
> 	"babel-loader": "8.0.6",
> 	...
> }
> ```
> 
> **System:** Environment Info: System: OS: Windows 8.1 6.3.9600 Binaries: Node: 12.14.1 Yarn: 2.0.0-rc.27 npm: 6.13.4 npmPackages: @storybook/cli: ^5.3.19 => 5.3.19 @storybook/html: 5.2.8 => 5.2.8
> ## **Additional context**

Thus with

```yaml
nodeLinker: node-modules

plugins:
  - path: .yarn/plugins/yarn-plugin-nixify.cjs
    spec: "https://raw.githubusercontent.com/stephank/yarn-plugin-nixify/main/dist/yarn-plugin-nixify.js"

yarnPath: .yarn/releases/yarn-3.2.0.cjs

packageExtensions:
  "babel-preset-react-app@*":
    peerDependencies:
      webpack: "*"
  "react-dev-utils@*":
    peerDependencies:
      webpack: "*"
      typescript: "*"
  "react-scripts@*":
    peerDependencies:
      typescript: "*"
  "org-web@*":
    peerDependencies:
      typescript: "*"
```

and
```console
yarn set version berry
yarn build
yarn dlx serve -s build
```

I was able to build.

**Comment by DanielDe** [comment]
Glad to hear you got this working! Thanks for the update
